lib/Client.js

```diff
@@ -167,6 +167,7 @@ class Client extends EventEmitter {
      * @param {Boolean} [options.voice.daveEncryption=true] Whether to enable DAVE E2EE encryption for voice connections
      * @param {Boolean} [options.voice.opusOnly=false] Whether to suppress the Opus encoder not found error or not
      * @param {Number} [options.voice.udpTimeout=10000] How long the IP discovery from the UDP should take until it times out
+     * @param {Number} [options.voice.decryptionFailureTolerance=24] How many consecutive decryption failures needed to reinitialize the E2EE session to recover
```

Comment:
This is probably meant to be 36 according to the rest of the code
```suggestion
     * @param {Number} [options.voice.decryptionFailureTolerance=36] How many consecutive decryption failures needed to reinitialize the E2EE session to recover
```